changelog/22.0/22.0.0/summary.md

```diff
@@ -15,6 +17,12 @@ These are the RPC changes made in this release -
 
 1. `GetTransactionInfo` RPC has been added to both `VtctldServer`, and `TabletManagerClient` interface. These RPCs are used to fecilitate the users in reading the state of an unresolved distributed transaction. This can be useful in debugging what went wrong and how to fix the problem.
 
+### <a id="deprecations-and-deletions"/>Deprecations and Deletions</a>
+
+#### <a id="vttablet-flags"/>Deprecated VTTablet Flags</a>
+
+- `twopc_enable` flag is deprecated, `twopc_abandonage` flag if set will be used to determine if 2PC is enabled or not.
```

Comment:
Are we really determining whether to allow 2PC based on `twopc_abandonage`? Ideally it should be controlled solely by transaction_mode on vtgate, and vttablets should be able to participate.

Reply:
I agree with Deepthi. There is no reason to say explicitly whether twopc is enabled or not. If the transaction mode is twopc then we run it, otherwise not.

Reply:
there's some prior art on how to change this over 2 releases. you should be able to find the relevant PRs in GH history.

Reply:
done [dfcd94c](https://github.com/vitessio/vitess/pull/17279/commits/dfcd94c763d1a9d553188b45c09b441bd32f36bc)